spec/spec_helpers/test_db.rb

```diff
@@ -4,66 +4,114 @@
 
 class TestDb
```

Comment:
This is reasonably generic; I think it would be good practice to namespace it. Same goes for `GlobalizeDb`.